powerauth-push-server/src/main/java/io/getlime/push/configuration/SecurityConfig.java

```diff
@@ -27,15 +28,15 @@
  */
 @Configuration
 @EnableWebSecurity
-public class SecurityConfig extends WebSecurityConfigurerAdapter {
+public class SecurityConfig {
 
     //TODO: Fix to contain secured setup.
 
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        http.authorizeRequests().anyRequest().anonymous();
-        http.httpBasic().disable();
-        http.csrf().disable();
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        return http.authorizeHttpRequests().anyRequest().anonymous()
+                .and().httpBasic().disable()
+                .csrf().disable().build();
```

Comment:
We may consider _lambda style_ to avoid `.and()`.
```suggestion
        return http
                .authorizeHttpRequests(authorize -> authorize
                        .anyRequest().anonymous())
                .httpBasic().disable()
                .csrf().disable()
                .build();
```

Reply:
Sure, why not.